lib/output.js

```diff
@@ -1444,17 +1449,21 @@ function OutputStream(options) {
         }
     });
     DEFPRINT(AST_ObjectKeyVal, function(self, output){
-        var print_as_shorthand = self.shorthand &&
+        var get_name = function(self) {
```

Comment:
I think the line above still has slightly more runtime overhead rather than using a regular local function:
```
function get_name(self) {
```